src/Regridder.jl

```diff
@@ -64,6 +65,33 @@ function reshape_cgll_sparse_to_field!(field::Fields.Field, in_array::Array, R)
     Spaces.dss!(Fields.field_values(field), topology, hspace.quadrature_style)
 end
 
+function reshape_cgll_sparse_to_field!(
```

Comment:
could you add comments explaining that this method is for 3d spaces and the one above is for 2d?